jbdevstudio/multiple/jbdevstudio-multiple.target

```diff
@@ -105,19 +105,19 @@
       <unit id="org.sonatype.m2e.mavenarchiver.feature.feature.group" version="0.17.0.201502101659"/>
     </location>
     <location includeAllPlatforms="false" includeMode="slicer" type="InstallableUnit" includeSource="true">
-      <repository location="http://download.jboss.org/jbosstools/updates/requirements/m2e/1.7.0.20160202-2119/"/>
+      <repository location="http://download.jboss.org/jbosstools/updates/requirements/m2e/1.7.0.20160504-0322/"/>
       <!--unit id="org.eclipse.m2e.feature.feature.group" version="1.7.0.20151110-2248"/>
       <unit id="org.eclipse.m2e.tests.common" version="1.7.0.20151110-2248"/-->
-      <unit id="org.eclipse.m2e.importer" version="1.7.0.20160202-2119"/>
+      <unit id="org.eclipse.m2e.importer" version="1.7.0.20160504-0322"/>
     </location>
 
     <!-- SimRel -->
     <location includeAllPlatforms="false" includeMode="slicer" type="InstallableUnit" includeSource="true">
-      <repository location="http://download.jboss.org/jbosstools/updates/requirements/neon/201602051000-M5/"/>
+      <repository location="http://download.jboss.org/jbosstools/updates/requirements/neon/20160505-1818-M7/"/>
```

Comment:
Good catch. The previous site contained incorrect metadata because my new jenkins job to fetch the mirror couldn't find ant-contrib and so didn't do the step that purged the OTDT stuff. PR updated.